hbp_knowledge/receptors/buckingham2009.bel

```diff
@@ -1203,22 +1296,27 @@ al., 2007) through the activation of nAChRs. It therefore
 seems that activation of nAChRs shifts the balance of
 APP processing away from beta-amyloidogenic to soluble
 APP production."
-
+SET Confidence = "Low"
 SET CellLine="PC12"
 
-p(HGNC:CHRNA7) -> a(CHEBI:nicotine)
-a(CHEBI:nicotine) -> sec(a(HBP:"sAPP-beta"))
+a(CHEBI:nicotine) -> p(HGNC:CHRNA7)
+#nicotine stimulates CHRNA7 and not vice versa
+p(HGNC:CHRNA7) -> sec(a(HBP:"sAPP-beta"))
```

Comment:
I would change it to
```suggestion
a(CHEBI:nicotine) -> act(p(HGNC:CHRNA7))
act(p(HGNC:CHRNA7)) -> sec(a(HBP:"sAPP-beta"))
```
as nicotine induces the secretion in a NACHR dependent way